manifests/plugin/abrt.pp

```diff
@@ -30,7 +33,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $group                   = $::foreman_proxy::plugin::abrt::params::group,
+  $version                 = $::foreman_proxy::plugin::version,
```

Comment:
I think this should default to 'undef' because kafo can't handle this. Also because I don't think you can refer to variables on a define in puppet.